ci/run

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -eou pipefail
 
+# Want to ensure that dev and master have been
+# checked out for testing purposes
+echo '--- Checkout branches'
+git submodule foreach "git checkout dev"
+git submodule foreach "git checkout master"
```

Comment:
note to self: This is a better way of doing the checkout https://github.com/radicle-dev/radicle-surf/pull/106/files#diff-a7b0a2dee0126cddf994326e705a91eaR24